prof.css

```diff
@@ -18,111 +18,109 @@ body{
   justify-content: center;
   width: 100%;
 }
-.top{
+.top__hello, .skills__position, .skills__language, .skills__tools, .startup__myemail, .startup__title, .network__title, .database__title{
```

Comment:
this is just not it, create a new class name color-red and give it to them in the html

Reply:
hala2 ma osset bad more about preference, bass i am the reviewer so yes bad change it